kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -33,18 +33,30 @@ def airflow_commands():
     type=click.Path(writable=True, resolve_path=True, file_okay=False),
     default="./airflow_dags/",
 )
+@click.option(
+    "-jd", "--jinja-dir", "jinja_template_dir", default=str(Path(__file__).parent)
+)
+@click.option("-j", "--jinja-file", "jinja_template", default="airflow_dag_template.j2")
 @click.pass_obj
 def create(
-    metadata: ProjectMetadata, pipeline_name, env, target_path
-):  # pylint: disable=too-many-locals
+    metadata: ProjectMetadata,
+    pipeline_name,
+    env,
+    target_path,
+    jinja_template_dir,
+    jinja_template,
+):  # pylint: disable=too-many-locals,too-many-arguments
     """Create an Airflow DAG for a project"""
-    loader = jinja2.FileSystemLoader(str(Path(__file__).parent))
+    loader = jinja2.FileSystemLoader(jinja_template_dir)
     jinja_env = jinja2.Environment(autoescape=True, loader=loader, lstrip_blocks=True)
     jinja_env.filters["slugify"] = slugify
-    template = jinja_env.get_template("airflow_dag_template.j2")
+    template = jinja_env.get_template(jinja_template)
 
     package_name = metadata.package_name
-    dag_filename = f"{package_name}_dag.py"
+    if pipeline_name != "__default__":
+        dag_filename = f"{package_name}_{pipeline_name}_dag.py"
+    else:
+        dag_filename = f"{package_name}_dag.py"
```

Comment:
I think it is a good idea to have the flexibility to the generate dag file name. I have two comments here.

1. It breaks the backward compatibility.
2. I don't have strong preference about the file name, but may as well just let user define whatever name they find meaningful?